tests/norm/test_add_rmsnorm_fp4_quant_cute_dsl.py

```
@@ -1283,5 +1312,900 @@ def test_auto_allocation_matches_preallocated(self, batch_size, hidden_size):
+        from flashinfer.cute_dsl import add_rmsnorm_fp4quant
+
+        block_size = 16
+        eps = 1e-6
+        torch.manual_seed(42)
+
+        x = torch.randn(batch_size, hidden_size, device="cuda", dtype=dtype)
+        r = torch.randn(batch_size, hidden_size, device="cuda", dtype=dtype)
+        weight = torch.randn(hidden_size, device="cuda", dtype=dtype)
+
+        # Call with output_both_sf_layouts=True
+        y_fp4, block_scale_swizzled, block_scale_unswizzled = add_rmsnorm_fp4quant(
```

Ruff (0.14.13)

Unpacked variable `y_fp4` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)